docs/multisigs/emergency-brakes.md

```diff
@@ -0,0 +1,155 @@
+# Emergency Brakes
+
+## 1.1 GateSeal Committee
+
+**Address:** [`0x8772E3a2D86B9347A2688f9bc1808A6d8917760C`](https://app.safe.global/transactions/queue?safe=eth:0x8772E3a2D86B9347A2688f9bc1808A6d8917760C)
+
+**Purpose of the multisig:** The GateSeal Committee can trigger GateSeal to pause WithdrawalQueueERC721 (pausing users' withdrawal requests), ValidatorExitBusOracle (pausing NOs withdrawal requests) or both smart contracts for 6 days. The right to pause is one-use only and automatically expires on 1 May 2024.
+
+**Quorum:** 3/6
+
+**Forum topic:** [Lido V2 GateSeal Committee](https://research.lido.fi/t/lido-v2-gateseal-committee/4561)
+
+**Snapshot:** [Voting for approval of new withdrawals mechanism and new modular architecture for Node Operators set](https://snapshot.org/#/lido-snapshot.eth/proposal/0x629b547c688dea536a4a5c5b42274894ac068df0b0278d173b4d7a68c8c4281d)
+
+**Aragon:** [Omnibus Vote #156](https://vote.lido.fi/vote/156)
```

Comment:
This vote shouldn't be described as an Omnibus vote, because it only contains one albeit complex item – the Lido protocol upgrade to V2. I suggest referring to it as just and Aragon Vote #156

Reply:
fixed